pkg/router/controller/contention.go

```diff
@@ -279,8 +292,11 @@ func ingressConditionsEqual(a, b []routev1.RouteIngressCondition) bool {
 	return true
 }
 
-// conditionsEqual compares two RouteIngressConditions, ignoring LastTransitionTime.
+// conditionsEqual compares two RouteIngressConditions, ignoring LastTransitionTime and any reason in ignoreIngressConditionReason..
```

Comment:
```suggestion
// conditionsEqual compares two RouteIngressConditions, ignoring LastTransitionTime and any reason in ignoreIngressConditionReason.
```
No need to fix this now, but maybe we can fix it in the next PR that touches the contention code.

Reply:
Thanks for the catch. Updated in https://github.com/openshift/router/commit/62daef459fde17c27007cd8d0ac6bc24209b2120